openapi-validator/src/main/java/com/networknt/openapi/RequestValidator.java

```diff
@@ -166,7 +176,7 @@ private Status validateQueryParameter(final HttpServerExchange exchange,
             if(queryParameter.getRequired()) {
                 return new Status(VALIDATOR_REQUEST_PARAMETER_QUERY_MISSING, queryParameter.getName(), openApiOperation.getPathString().original());
             }
-        } else {
+        } else if (queryParameterValues.size() < 2) {
 
```

Comment:
Can you add a comment explaining this check please?

Reply:
Sure, I will do that.